.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules/
 examples/node/svgdump/
 examples/node/pdf2png/*.png
 package-lock.json
+.idea
```

Comment:
This shouldn't be here, please remove it.

Reply:
Sure will do that in my next commit